ci/run.sh

```diff
@@ -13,4 +13,5 @@ clone_tests_repo
 
 pushd ${tests_repo_dir}
 .ci/run.sh
+tracing/test-agent-shutdown.sh
```

Comment:
OOI: This is for the GHA checks only, right? Because it would be great to test on all architectures.